ibm/data_source_ibm_is_operating_systems.go

```diff
@@ -0,0 +1,124 @@
+// Copyright IBM Corp. 2017, 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+const (
+	isOperatingSystems = "operating_systems"
+)
+
+func dataSourceIBMISOperatingSystems() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceIBMISOperatingSystemsRead,
+
+		Schema: map[string]*schema.Schema{
+			isOperatingSystems: {
+				Type:        schema.TypeList,
+				Description: "List of operating systems",
+				Computed:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						isOperatingSystemName: {
+							Type:     schema.TypeString,
+							Required: true,
```

Comment:
Attribute cannpt be required if parent is computed
Can you please Mark it as Computed..
and also all schema attributes require description